templates/bridgeserver2.yaml

```diff
@@ -276,26 +309,26 @@ Resources:
           OptionName: aws.key
           Value: !Ref AWSIAMBridgeServer2ServiceUserAccessKey
         - Namespace: 'aws:elasticbeanstalk:application:environment'
-          OptionName: aws.key.consents
-          Value: !Ref AWSIAMBridgeServer2ServiceUserAccessKey
+          OptionName: aws.secret.key
+          Value: !GetAtt AWSIAMBridgeServer2ServiceUserAccessKey.SecretAccessKey
         - Namespace: 'aws:elasticbeanstalk:application:environment'
-          OptionName: aws.key.upload
-          Value: !Ref AWSIAMBridgeServer2ServiceUserAccessKey
+          OptionName: admin.email
+          Value: !Ref BootstrapAdminEmail
         - Namespace: 'aws:elasticbeanstalk:application:environment'
-          OptionName: aws.key.upload.cms
-          Value: !Ref AWSIAMBridgeServer2ServiceUserAccessKey
+          OptionName: admin.password
+          Value: !Ref BootstrapAdminPassword
```

Comment:
@DwayneJengSage Sorry I missed this in my review.  These secrets are already in the SSM so you don't need to set it as an ENV VAR.  You just need to give the EC2 instances launched by beanstalk access to the SSM parameters that contain these secrets.  Then your app can get the secrets directly from the SSM instead of from an ENV VAR.  That's a more secure approach.